<html>
<head><meta charset="utf-8"><title>failed upgrade 1.8.1 -&gt; 1.9.1 · production help · Zulip Chat Archive</title></head>
<h2>Stream: <a href="https://rht.github.io/czo-archive/stream/31-production-help/index.html">production help</a></h2>
<h3>Topic: <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html">failed upgrade 1.8.1 -&gt; 1.9.1</a></h3>

<hr>

<base href="https://chat.zulip.org">

<head><link href="https://rht.github.io/czo-archive/style.css" rel="stylesheet"></head>

<a name="672380"></a>
<h4><a href="https://chat.zulip.org#narrow/stream/31-production%20help/topic/failed%20upgrade%201.8.1%20-%3E%201.9.1/near/672380" class="zl"><img src="https://rht.github.io/czo-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Will Monge <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html#672380">(Dec 16 2018 at 05:13)</a>:</h4>
<p>Sorry, ran into some problems when trying to upgrade my zulip server (currently 1.8.1) to the latest 1.9.1.</p>
<p>When running:<br>
<code>sudo /home/zulip/deployments/current/scripts/upgrade-zulip ~/zulip-tarballs/zs-1.9.1/zulip-server-latest.tar.gz</code></p>
<p>I got the following:</p>
<div class="codehilite"><pre><span></span>E: mkinitramfs failure find 141 cpio 141 gzip 1
update-initramfs: failed for /boot/initrd.img-4.4.0-139-generic with 1.
dpkg: error processing package initramfs-tools (--configure):
 subprocess installed post-installation script returned error exit status 1
No apport report written because MaxReports is reached already
                                                              Errors were encountered while processing:
 linux-firmware
 linux-image-extra-4.4.0-139-generic
 linux-image-generic
 linux-generic
 initramfs-tools
E: Sub-process /usr/bin/dpkg returned an error code (1)
Traceback (most recent call last):
  File &quot;/home/zulip/deployments/2018-12-15-21-17-13/scripts/lib/upgrade-zulip-stage-2&quot;, line 54, in &lt;module&gt;
    subprocess.check_call([&quot;apt-get&quot;, &quot;-y&quot;, &quot;upgrade&quot;])
  File &quot;/usr/lib/python3.5/subprocess.py&quot;, line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command &#39;[&#39;apt-get&#39;, &#39;-y&#39;, &#39;upgrade&#39;]&#39; returned non-zero exit status 100
Traceback (most recent call last):
  File &quot;/home/zulip/deployments/current/scripts/lib/upgrade-zulip&quot;, line 52, in &lt;module&gt;
    subprocess.check_call([os.path.abspath(&quot;./scripts/lib/upgrade-zulip-stage-2&quot;), deploy_path])
  File &quot;/usr/lib/python3.5/subprocess.py&quot;, line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command &#39;[&#39;/home/zulip/deployments/2018-12-15-21-17-13/scripts/lib/upgrade-zulip-stage-2&#39;, &#39;/home/zulip/deployments/2018-12-15-21-17-13&#39;]&#39; returned non-zero exit status 1
</pre></div>


<p>Have been doing some digging through older failed upgrade topics here and seem to have come across a similar problem some time ago: <a href="#narrow/stream/31-production-help/subject/upgrade.20failing" title="#narrow/stream/31-production-help/subject/upgrade.20failing">https://chat.zulip.org/#narrow/stream/31-production-help/subject/upgrade.20failing</a></p>



<a name="672381"></a>
<h4><a href="https://chat.zulip.org#narrow/stream/31-production%20help/topic/failed%20upgrade%201.8.1%20-%3E%201.9.1/near/672381" class="zl"><img src="https://rht.github.io/czo-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Rein Zustand <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html#672381">(Dec 16 2018 at 05:22)</a>:</h4>
<p>Looks like a dist upgrade fail problem in general. What is the error message before "E: mkinitramfs failure find 141 cpio 141 gzip 1"?</p>



<a name="672382"></a>
<h4><a href="https://chat.zulip.org#narrow/stream/31-production%20help/topic/failed%20upgrade%201.8.1%20-%3E%201.9.1/near/672382" class="zl"><img src="https://rht.github.io/czo-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Will Monge <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html#672382">(Dec 16 2018 at 05:24)</a>:</h4>
<p>Ah thanks! It seems the updates are failing because I'm running our of disk space:<br>
<code>gzip: stdout: No space left on device</code></p>



<a name="672383"></a>
<h4><a href="https://chat.zulip.org#narrow/stream/31-production%20help/topic/failed%20upgrade%201.8.1%20-%3E%201.9.1/near/672383" class="zl"><img src="https://rht.github.io/czo-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Will Monge <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html#672383">(Dec 16 2018 at 05:25)</a>:</h4>
<p>thanks, will look into this! Sorry, for failing to see the line right there <span class="emoji emoji-1f605" title="sweat smile">:sweat_smile:</span></p>



<a name="672384"></a>
<h4><a href="https://chat.zulip.org#narrow/stream/31-production%20help/topic/failed%20upgrade%201.8.1%20-%3E%201.9.1/near/672384" class="zl"><img src="https://rht.github.io/czo-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Rein Zustand <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html#672384">(Dec 16 2018 at 05:26)</a>:</h4>
<p><span class="user-mention" data-user-id="8187">@Will Monge</span> you will have to clean up old kernels, <a href="https://help.ubuntu.com/community/RemoveOldKernels" target="_blank" title="https://help.ubuntu.com/community/RemoveOldKernels">https://help.ubuntu.com/community/RemoveOldKernels</a>.</p>



<a name="672386"></a>
<h4><a href="https://chat.zulip.org#narrow/stream/31-production%20help/topic/failed%20upgrade%201.8.1%20-%3E%201.9.1/near/672386" class="zl"><img src="https://rht.github.io/czo-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Will Monge <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html#672386">(Dec 16 2018 at 05:34)</a>:</h4>
<p>yup, just ran <code>sudo apt-get autoremove --purge</code> as specified in the link above and was able to clear my <code>/boot</code> drive considerably. Running the upgrade again, hopefully things should proceed smoothly now. Thanks <span class="user-mention" data-user-id="2328">@Rein Zustand</span> !</p>



<a name="672387"></a>
<h4><a href="https://chat.zulip.org#narrow/stream/31-production%20help/topic/failed%20upgrade%201.8.1%20-%3E%201.9.1/near/672387" class="zl"><img src="https://rht.github.io/czo-archive/assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Will Monge <a href="https://rht.github.io/czo-archive/stream/31-production-help/topic/failed.20upgrade.201.2E8.2E1.20-.3E.201.2E9.2E1.html#672387">(Dec 16 2018 at 05:45)</a>:</h4>
<p>Yup, now running the new and shiny 1.9.1. <span class="user-mention" data-user-id="2328">@Rein Zustand</span> thanks again! I am still quite new to sysadmining, so didn't know I had to clear the drive of old kernels. Thanks!</p>



<hr><p>Last updated: Oct 02 2023 at 04:38 UTC</p>
</html>